src/tuesday/core/plotting/powerspectra.py

```diff
@@ -0,0 +1,368 @@
+"""Plotting functions for 1D and 2D power spectra."""
+
+import astropy.units as un
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.cosmology.units import littleh
+from matplotlib import rcParams
+from scipy.ndimage import gaussian_filter
+
+
+def plot_1d_power_spectrum(  # noqa: C901
+    wavemodes: un.Quantity,
+    power_spectrum: un.Quantity,
+    fig: plt.Figure | None = None,
+    ax: plt.Axes | None = None,
+    title: str | None = None,
+    xlabel: str | None = None,
+    ylabel: str | None = None,
+    colors: list | None = None,
+    log: list[bool] | None = None,
+    fontsize: float | None = 16,
+    labels: list | None = None,
+    smooth: float | bool = False,
+    leg_kwargs: dict | None = None,
+) -> tuple[plt.Figure, plt.Axes]:
+    """
+    Plot a 1D power spectrum.
+
+    Parameters
+    ----------
+    wavemodes : np.ndarray
+        Wavemodes corresponding to the power spectrum.
```

Comment:
Incorrect typing info in the docstring here, and maybe just tell the user what units are OK.